docs/coding-style.md

```diff
@@ -194,13 +228,46 @@ Variable name `sut` (System Under Test) is recommended in unit tests that really
 * Var pattern `o is { P: var p }` can be used only where `o` can be `null` and `p` is used at least 3 times.
 * Each compiler directive outside method body (namely `#if`/`#endif`) should be preceded and followed by an empty line.
 * Do not use `nullable`.
+* Avoid single-use variables, unless they really bring readability value.
+* Use file-scoped namespaces.
+* Tested variable is on the left, like `iterated == searchParameter` (where `iterated` is the tested variable)
+* If a string is multiline, use raw string literals, indented one tab-in from the declaration:
+```
+var raw = """
+    hello
+    my friend
+	""";
+```
+* Always use multi-line initializers for collection and objects, e.g.:
+```
+var thingy = new Thingy
+{
+	x = "hello",
+	y = 42,
+}
+
+var collection = new Dictionary<string, int>
+{
+	{ "hey" : 1 },
+	{ "there": 42 },
+}
+```
+* FIXME - align on how to use collection initializers int[] x = [ 1, 2, 3 ] or old style (see [slack discussion](https://sonarsource.slack.com/archives/C01H2B58DE1/p1697103918957899?thread_ts=1696951023.295859&cid=C01H2B58DE1))
+
+### Unit Tests
+* VerifierBuilder.AddSnippet should not be used to assert compliant/noncomplaint test cases. Move it to a TestCases file.
+
 
 ## Comments
 
 * Code should contain as few comments as necessary in favor of well-named members and variables.
 * Comments should generally be on separate lines.
 * Comments on the same line with code are acceptable for short lines of code and short comments.
 * Documentation comments for abstract methods and their implementations should be placed only on the abstract method, to avoid duplication. _When reading the implementation, the IDE offers the tooling to peek in the base class and read the method comment._
+* Avoid using comments for "Arrange, Act, Assert" in UTs, unless the test is complex.
+* Use single-line comments. Exception: `Internal /* for testing */ void Something()`.
```

Comment:
The naming here is confusing since this is a single line.
```suggestion
* Use only double-slash comments. Exception: `Internal /* for testing */ void Something()`.
```

Reply:
Or maybe rather "Do not use `/* ... */`."